queries/java.scm

```diff
@@ -326,3 +326,34 @@
   body: (_) @value
   (#not-type? @value block)
 ) @_.domain
+
+;;!! public Map<int, int> foo;
+;;!         ^^^^^^^^^^^^^
+;;!  -------------------------
+(field_declaration
+  type: (_) @type
+) @_.domain
+
+;;!! class MyClass { }
+;;!                 ^
+(class_body
+  .
+  "{" @type.iteration.start.endOf
+  "}" @type.iteration.end.startOf
+  .
+)
+
+;;!! public Map<int, int> foo;
+;;!             ^^^  ^^^
+(type_arguments
+  (_) @type
+)
```

Comment:
shall we add a facet for this?

Reply:
Probably. What should we call that? `type.typeArguments`?

Reply:
Works for me

Reply:
done